flow/test/test_helper.sh

```diff
@@ -84,14 +84,13 @@ case $DESIGN_NAME in
     RUN_AUTOTUNER=1
     ;;
 esac
-case $PLATFORM in
-     "asap7" | "sky130hd" | "ihp-sg13g2" )
-      # Keep RUN_AUTOTUNER enabled only for these platforms
-      ;;
-     *)
-      RUN_AUTOTUNER=0
-      ;;
-esac
+if [ $RUN_AUTOTUNER -eq 1 ]; then
+  case $PLATFORM in
+       "gf180" | "nangate45" | "sky130hd_fakestack" | "sky130hs")
+        RUN_AUTOTUNER=0
+        ;;
+  esac
+fi
```

Comment:
Why change this? We need to ensure that AT will not run on private PDKs for now.